src/ralph/scan/plugins/ssh_cisco_asa.py

```diff
@@ -110,7 +110,8 @@ def scan_address(ip_address, **kwargs):
         ssh.close()
     pairs = parse.pairs(lines=[line.strip() for line in lines])
     sn = pairs.get('Serial Number', None)
-    model, ram, cpu = pairs['Hardware'].split(',')
+    # hardware = model, ram, cpu, corenum*
+    hardware = pairs['Hardware'].split(',')
```

Comment:
It is not possible. Some devices have 3 variables (model, ram, cpu) and some have 4 variables (model, ram, cpu, corenum). I can remove comment.